index.html

```diff
@@ -38,7 +38,133 @@ <h2>LET'S CONNECT</h2>
             </section>
         </main>
         </div>
-    </div>
-    
+        
+    </div>  
+</div>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio.png" alt="">
+    <h1>Tonic</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
```

Comment:
You have done good work in organizing your code for sections. But I want to clear a misunderstanding regarding `section`. So section is like, your `infoText` section, `work-section` or `about-section`. So since you were designing your work section, so work section would be 1 section that would contain multiple `work-cards`.
So, kindly use one section for work section.